modules/tools/calibration/process.py

```diff
@@ -48,12 +48,12 @@ def get_start_index(data):
     if np.all(data['vehicle_speed'] == 0):
         return 0
 
-    start_ind = np.where(data['brake_percentage'] == 40)[0][0]
+    start_ind = np.where(data['vehicle_speed'] == 0.0)[0][0]
```

Comment:
Not safe to compare `equal` on float numbers.

Reply:
modified as your require